src/cljs/rems/table.cljs

```diff
@@ -295,29 +329,52 @@
                                                                                        " > tbody > tr[data-row='" next-row "'] > td"))))}
                 (text-format :t.table/show-all-n-rows (count rows))]]]])]]))
 
+;;; guide
+
+(def example-selected-rows (reagent/atom nil))
+
+(rf/reg-sub ::empty-table-rows (fn [_ _] []))
+
+(defn- example-commands [text]
+  {:td [:td.commands [:button.btn.btn-primary {:on-click #(do (js/alert (str "View " text)) (.stopPropagation %))} "View"]]})
+
+(rf/reg-sub
+ ::example-table-rows
+ (fn [_ _]
+   [{:key 1
+     :first-name {:value "Cody"}
+     :last-name {:value "Turner"}
+     :commands (example-commands "Cody")}
+    {:key 2
+     :first-name {:value "Melanie"}
+     :last-name {:value "Palmer"}
+     :commands (example-commands "Melanie")}
+    {:key 3
+     :first-name {:value "Henry"}
+     :last-name {:value "Herring"}
+     :commands (example-commands "Henry")}]))
+
+(rf/reg-sub
+ ::example-rich-table-rows
+ (fn [_ _]
+   [{:key 1
+     :team {:display-value "Team Hawks"
+            :filter-value "hawks"
+            :sort-value "0000hawks"}
+     :points {:value 3
+              :display-value "-> 3 <-"}}
+    {:key 2
+     :team {:value "Eagles"
+            :td [:td.eagles-are-best [:em "Eagles"]]}
+     :points {:value 4}}
+    {:key 3
+     :team {:value "Ravens"}
+     :points {:value 0}}]))
+
 (defn guide []
   [:div
    (namespace-info rems.table)
    (component-info table)
-   ;; slight abuse of example macro, but it works since reg-sub returns a fn which reagent doesn't render
-   (example "data for examples"
-            [:p "Data is provided to the table component as a subscription"]
```

Comment:
wasn't it nicer when the subscription data was shown in the guide too?

Reply:
You are correct that it's nicer to show it. The use of subscription as parameter makes it difficult here. I'll think about it if there is another way.